Error during `make install`

I was running `make install` as described in the readme and the following error showed up:

```bash
avanderlee@Antoines-MBP-2 CreateAPI % make install
swift build --disable-sandbox -c release 
warning: Usage of /Users/avanderlee/Library/org.swift.swiftpm/collections.json has been deprecated. Please delete it and use the new /Users/avanderlee/Library/org.swift.swiftpm/configuration/collections.json instead.
Building for production...
Build complete! (0.15s)
warning: Usage of /Users/avanderlee/Library/org.swift.swiftpm/collections.json has been deprecated. Please delete it and use the new /Users/avanderlee/Library/org.swift.swiftpm/configuration/collections.json instead.
mkdir -p /usr/local/bin
sudo cp -f /Users/avanderlee/Developer/GIT-Projects/Forks/CreateAPI/.build/arm64-apple-macosx/release/CreateAPI /usr/local/bin/create-api
cp: /Users/avanderlee/Developer/GIT-Projects/Forks/CreateAPI/.build/arm64-apple-macosx/release/CreateAPI: No such file or directory
make: *** [install] Error 1
avanderlee@Antoines-MBP-2 CreateAPI % 
```

I'm using `Xcode 13.4.1 (13F100)` and I'm on this machine:

![CleanShot 2022-07-08 at 11 07 06@2x](https://user-images.githubusercontent.com/4329185/177958331-6bc20cf6-bbb6-4e4e-8104-14d23c0aa534.jpg)

--- a/Makefile
+++ b/Makefile
@@ -1,14 +1,15 @@
-export EXECUTABLE_NAME = CreateAPI
-PREFIX = /usr/local
-INSTALL_PATH = $(PREFIX)/bin/create-api
-CURRENT_PATH = $(PWD)
-REPO = https://github.com/CreateAPI/$(EXECUTABLE_NAME)
-SWIFT_BUILD_FLAGS = --disable-sandbox -c release # --arch arm64 # --arch x86_64
-EXECUTABLE_PATH = $(shell swift build $(SWIFT_BUILD_FLAGS) --show-bin-path)/$(EXECUTABLE_NAME)
+SWIFT_BUILD_FLAGS = --disable-sandbox -c release
+BINARIES_PATH = /usr/local/bin
+EXECUTABLE_PATH = $(shell swift build $(SWIFT_BUILD_FLAGS) --show-bin-path)/create-api
+
+.PHONY: build install uninstall
 
 build:
 	swift build $(SWIFT_BUILD_FLAGS)
 
 install: build
-	mkdir -p $(PREFIX)/bin
-	sudo cp -f $(EXECUTABLE_PATH) $(INSTALL_PATH)
+	sudo install -d "$(BINARIES_PATH)"
+	sudo install "$(EXECUTABLE_PATH)" "$(BINARIES_PATH)"
+
+uninstall:
+	sudo rm -f "$(BINARIES_PATH)/create-api"

